util/statvar_dcid_generator.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+import re
+
+
+class StatVarDcidGenerator:
+    """Helper class to generate dcid for a statistical variable."""
+    STAT_TYPE_LOOKUP = {'meanValue': 'mean', 'medianValue': 'median'}
+
+    def __init__(self):
+        pass
+
+    def capitalize_word(self, word):
+        """Changes the first letter of a word to upper case."""
+        if (len(word) != 0):
+            return (word[0].upper() + word[1:])
+        else:
+            return None
+
+    def append_to_dcid(self, dcid, value, end_underscore=False):
+        """Append a value to a dcid."""
+        dcid += self.capitalize_word(value)
+        if (end_underscore == True):
+            dcid += '_'
+
+        return dcid
```

Comment:
Probably collect things in a list and do a final '_'.join(dcid_parts)

Reply:
Done.